python/cudf/cudf/core/dataframe.py

```diff
@@ -4429,13 +4429,7 @@ def scatter_by_map(self, map_index, map_size=None, keep_index=True):
         return tables
 
     def repeat(self, repeats, axis=None):
```

Comment:
Need to assert the axis here still, instead of using `self._repeat` it should be `super().repeat`